src/components/CreateAlias.vue

```diff
@@ -235,15 +255,22 @@ export default {
 }
 
 .intro-text {
-  max-width: 500px;
+  line-height: 34px;
+  max-width: 973px;
   color: white;
   margin: auto;
+  font-size: 20px;
+  font-weight: 300;
 }
 
 @media (max-width: 625px) {
   /* At this width we want to start shrinking. 80% of 625 = 500 */
   .intro-text {
     max-width: 80vw;
+    font-size:18px;
+    line-height:27px;
+    margin-top:-2%;
```

Comment:
Seemed odd we nee to use negative magins and line-hight changes here.

Reply:
fixed (delete)